services/payment/src/EShopOnAbp.PaymentService.Domain/EventHandlers/OrderCancelledEventHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using EShopOnAbp.OrderingService.Orders;
+using EShopOnAbp.PaymentService.PaymentRequests;
+using Serilog;
+using System.Threading.Tasks;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.EventBus.Distributed;
+
+
+namespace EShopOnAbp.PaymentService.EventHandlers
+{
+    public class OrderCancelledEventHandler : IDistributedEventHandler<OrderCancelledEto>, ITransientDependency
+    {
+        private readonly IPaymentRequestRepository _paymentRepository;
+
+        public OrderCancelledEventHandler(IPaymentRequestRepository paymenRepository)
+        {
+            _paymentRepository = paymenRepository;
+        }
+
+        public async Task HandleEventAsync(OrderCancelledEto eventData)
+        {
+            var payment = await _paymentRepository.GetAsync(eventData.Buyer.BuyerId.GetValueOrDefault());
+            Log.Information($"Cancelled the order: {payment.OrderId} payment:{payment.Id}");
+        }
+    }
```

Comment:
Buyer can have multiple orders. Which order to refund can not be known by using the buyer.

This handler should be redesigned based on `OrderCancelledEto` review to use PaymentRequestId.